src/tests/ftest/rebuild/pool_destroy_race.py

```diff
@@ -40,13 +40,15 @@ def test_pool_destroy_with_io(self):
         """
         # set params
         targets = self.params.get("targets", "/run/server_config/*/0/*")
-        rank = self.params.get("rank_to_kill", "/run/testparams/*")
         engines_per_host = self.params.get("engines_per_host",
                                            "/run/server_config/*")
 
         # create pool
         self.create_pool()
 
+        # get a non-svc rank to kill
+        rank = self.pool.get_storage_ranks(num_ranks=1)[0]
```

Comment:
(pylint-unexpected-keyword-arg) Unexpected keyword argument 'num_ranks' in method call